<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>758</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>641</width>
      <height>401</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Euphemia UCAS</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>Home</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>261</width>
        <height>151</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>81</width>
         <height>71</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="photos.qrc">:/icons/home_download.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>60</y>
         <width>121</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Euphemia UCAS</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Download any type of file</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>20</y>
        <width>261</width>
        <height>151</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>81</width>
         <height>71</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="photos.qrc">:/icons/youtube.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>50</y>
         <width>121</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Euphemia UCAS</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Youtube video download</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>200</y>
        <width>261</width>
        <height>151</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>81</width>
         <height>71</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="photos.qrc">:/icons/home_youtube_playlist.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>50</y>
         <width>121</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Euphemia UCAS</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Download YouTube playlist</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>200</y>
        <width>261</width>
        <height>151</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>81</width>
         <height>81</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="photos.qrc">:/icons/home_settings.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_17">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>50</y>
         <width>121</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Euphemia UCAS</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>App themes &amp; settings</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Youtube</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>671</width>
        <height>401</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>One Video</string>
       </attribute>
       <widget class="QLineEdit" name="lineEdit_3">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>50</y>
          <width>391</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Euphemia UCAS</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="placeholderText">
         <string>Enter video URL</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_4">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>100</y>
          <width>391</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Euphemia UCAS</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="placeholderText">
         <string>Choose storage location</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_3">
        <property name="geometry">
         <rect>
          <x>500</x>
          <y>90</y>
          <width>61</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="photos.qrc">
          <normaloff>:/icons/browse.png</normaloff>:/icons/browse.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>33</width>
          <height>33</height>
         </size>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_4">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>310</y>
          <width>171</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Euphemia UCAS</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Start Download</string>
        </property>
       </widget>
       <widget class="QProgressBar" name="progressBar_2">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>210</y>
          <width>451</width>
          <height>23</height>
         </rect>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>160</y>
          <width>131</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Euphemia UCAS</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Video Quality</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>160</y>
          <width>251</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Euphemia UCAS</family>
          <pointsize>18</pointsize>
         </font>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_5">
        <property name="geometry">
         <rect>
          <x>500</x>
          <y>50</y>
          <width>61</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="photos.qrc">
          <normaloff>:/icons/check.png</normaloff>:/icons/check.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>33</width>
          <height>33</height>
         </size>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>260</y>
          <width>221</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Euphemia UCAS</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Playlist</string>
       </attribute>
       <widget class="QLineEdit" name="lineEdit_5">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>20</y>
          <width>521</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Euphemia UCAS</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="placeholderText">
         <string>Enter download URL</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_6">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>70</y>
          <width>461</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Euphemia UCAS</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="placeholderText">
         <string>Browse storage location</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_6">
        <property name="geometry">
         <rect>
          <x>510</x>
          <y>60</y>
          <width>61</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="photos.qrc">
          <normaloff>:/icons/browse.png</normaloff>:/icons/browse.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>33</width>
          <height>33</height>
         </size>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>130</y>
          <width>131</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Euphemia UCAS</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Video Quality</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox_2">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>120</y>
          <width>291</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Euphemia UCAS</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <item>
         <property name="text">
          <string>1080</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>720</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>480</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>360</string>
         </property>
        </item>
       </widget>
       <widget class="QProgressBar" name="progressBar_3">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>210</y>
          <width>521</width>
          <height>23</height>
         </rect>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_7">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>280</y>
          <width>181</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Euphemia UCAS</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Start Download</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>180</y>
          <width>91</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Current Video</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>180</y>
          <width>91</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Full Playlist</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>500</x>
          <y>210</y>
          <width>60</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Euphemia UCAS</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>215</y>
          <width>60</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Euphemia UCAS</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>240</y>
          <width>191</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Euphemia UCAS</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Download</string>
     </attribute>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>230</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Euphemia UCAS</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(128, 128, 128);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Download</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>70</y>
        <width>431</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Euphemia UCAS</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="placeholderText">
       <string>Enter download link</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>120</y>
        <width>381</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Euphemia UCAS</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="placeholderText">
       <string>Select storage location</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>120</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="photos.qrc">
        <normaloff>:/icons/browse.png</normaloff>:/icons/browse.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>33</width>
        <height>33</height>
       </size>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>170</y>
        <width>431</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Settings</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_12">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>141</width>
        <height>141</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(50,50,50);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_13">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>90</y>
        <width>141</width>
        <height>141</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(25,35,45);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_14">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>90</y>
        <width>141</width>
        <height>141</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(50,50,50);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>10</y>
        <width>241</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Euphemia UCAS</family>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Choose your theme</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_15">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>90</y>
        <width>141</width>
        <height>141</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="photos.qrc">
      <normaloff>:/icons/home.png</normaloff>:/icons/home.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="photos.qrc">
      <normaloff>:/icons/settings.png</normaloff>:/icons/settings.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="photos.qrc">
      <normaloff>:/icons/youtube.png</normaloff>:/icons/youtube.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_11">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="photos.qrc">
      <normaloff>:/icons/download.png</normaloff>:/icons/download.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>758</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="photos.qrc"/>
 </resources>
 <connections/>
</ui>
